ext/xmlrpc/tests/bug75282.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Bug #75282 (xmlrpc_encode_request() crashes)
+--SKIPIF--
+<?php
+if (!extension_loaded('xmlrpc')) die('skip xmlrpc extension not available');
+?>
+--FILE--
+<?php
+echo xmlrpc_encode_request('func', 'text', []);
+?>
+===DONE===
+--EXPECT--
+<?xml version="1.0" encoding="iso-8859-1"?>
+<methodCall>
+<methodName>func</methodName>
+<params>
+ <param>
+  <value>
+   <string>text</string>
+  </value>
+ </param>
+</params>
+</methodCall>
+===DONE===
```

Comment:
This test does not look particularly useful. Any of the other xmlrpc_encode_request tests already tests for this issue, if the extension is compiled in a way that exhibits the problem.

Reply:
Indeed. Thanks!